src/doc/man/generated_txt/cargo-fetch.txt

```diff
@@ -82,21 +82,6 @@ OPTIONS
            Cargo.lock file is up-to-date (such as a CI build) or want to avoid
            network access.
 
-       --offline
-           Prevents Cargo from accessing the network for any reason. Without
-           this flag, Cargo will stop with an error if it needs to access the
-           network and the network is not available. With this flag, Cargo will
-           attempt to proceed without the network if possible.
-
-           Beware that this may result in different dependency resolution than
```

Comment:
> Beware that this may result in different dependency resolution than
Comment online mode

Haven't tested it. If this statement is true, we may want to keep this, as it makes a huge difference between with and without `--offline`.

Reply:
> See the cargo-fetch(1)
           command to download dependencies before going offline.
           
I may have taken too big a step, and the original problem may have just been to remove this